src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/mvn.cpp

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/mvn.hpp"
+#include "shared_test_classes/single_layer/mvn.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace MVN {
+
+const std::vector<bool> normalizeVariance = {
+       true
+};
+
+std::vector<ElementType> inpPrc = {
+        ElementType::i8,
+        ElementType::f32,
+};
+std::vector<ElementType> outPrc = {
+        ElementType::f32,
+};
+
+std::vector<CPUSpecificParams> cpuParams_4D = {
+        CPUSpecificParams({nchw}, {nchw}, {}, {}),
+        CPUSpecificParams({nhwc}, {nhwc}, {}, {}),
+};
+
+std::vector<CPUSpecificParams> cpuParams_5D = {
+        CPUSpecificParams({ncdhw}, {ncdhw}, {}, {}),
+        CPUSpecificParams({ndhwc}, {ndhwc}, {}, {}),
+};
+
+std::vector<fusingSpecificParams> fusingParamsSet {
+        emptyFusingSpec,
+};
+
+std::vector<fusingSpecificParams> fusingParamsSetStaticShape {
+        emptyFusingSpec,
+};
+
+const auto Mvn3D = ::testing::Combine(
+       ::testing::Combine(
+           ::testing::ValuesIn(inputShapes_3D()),
+           ::testing::Values(ElementType::f32),
+           ::testing::ValuesIn(emptyReductionAxes()),
+           ::testing::ValuesIn(acrossChannels()),
+           ::testing::ValuesIn(normalizeVariance),
+           ::testing::ValuesIn(epsilon())),
+       ::testing::Values(emptyCPUSpec),
+       ::testing::ValuesIn(fusingParamsSet),
+       ::testing::ValuesIn(inpPrc),
+       ::testing::ValuesIn(outPrc));
+
+INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_Mvn3D, MvnLayerCPUTest, Mvn3D, MvnLayerCPUTest::getTestCaseName);
+
+const auto Mvn4D = ::testing::Combine(
+       ::testing::Combine(
+               ::testing::ValuesIn(inputShapes_4D()),
+               ::testing::Values(ElementType::f32),
+               ::testing::ValuesIn(emptyReductionAxes()),
+               ::testing::Values(true),
+               ::testing::ValuesIn(normalizeVariance),
+               ::testing::ValuesIn(epsilon())),
+       ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D)),
+       ::testing::ValuesIn(fusingParamsSet),
+       ::testing::ValuesIn(inpPrc),
+       ::testing::ValuesIn(outPrc));
+
+INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_Mvn4D, MvnLayerCPUTest, Mvn4D, MvnLayerCPUTest::getTestCaseName);
+
+const auto Mvn5D = ::testing::Combine(
+       ::testing::Combine(
+               ::testing::ValuesIn(inputShapes_5D()),
+               ::testing::Values(ElementType::f32),
+               ::testing::ValuesIn(emptyReductionAxes()),
+               ::testing::Values(true),
```

Comment:
Is the limitation applicable for the 5D as well?
The check in the source code seems to be applicable only for 4D tensor.

Reply:
ACL really limits MVN to handle up to 4 rank.
Dmitry suggested to implement WA: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/executors/acl/acl_mvn.cpp#L22-L47

ACL has 2D input shape [Y, X] and actual normalization is done over X axis, while Y serves as "batch". We can translate any 2+ dimension MVN shape into ACL representation using the following rules (example for 4D):

NCHW layout, across_channels == false; : Y = NC, X = HW
NCHW layout, across_channels == true; : Y = N, X = CHW
NHWC layout, across_channels == false; : unsupported
NHWC layout, across_channels == true; : Y = N, X = HWC